app/serializers/appointment_serializer.rb

```diff
@@ -8,4 +8,264 @@ class AppointmentSerializer < ActiveModel::Serializer
   def id
     nil
   end
+  # long lines
```

Comment:
Does this chunk need to be removed (Lines 11-270)?  Or maybe you did already? I'm guessing it was for testing purposes. Otherwise, looks good! I like the message details 👍

Reply:
Yep just for testing 👍